### ISSUE TITLE
Error dialog and stuck program

### ISSUE THREAD

**Issue by jpcima** [rerrahkr/BambooTracker#102]
After performing the steps, the program will meet an out of range condition.
The exception will trigger an error dialog, and when closed it will enter a new error, repeat and it will pop up other dialogs indefinitely.

1. load `demos/Underwater Ruins OPNA.btm`
2. click on instrument `0A: Hold+Decay Short`
3. click the top row of `SSG1`
4. play a note on it by keyboard

It will pop up 4(!) error dialogs at once: "An unknown error occured.".

branch master @ 2cbdbd191242d15c7f738c7a7849e44d9c286eb2

Breakpoint at `__cxa_throw`:
```
#0  __cxxabiv1::__cxa_throw (obj=obj@entry=0x7fffec007dc0, tinfo=0x7ffff69af038 <typeinfo for std::out_of_range>, 
    dest=0x7ffff68d2ba0 <std::out_of_range::~out_of_range()>)
    at /build/gcc/src/gcc/libstdc++-v3/libsupc++/eh_throw.cc:77
#1  0x00007ffff68b87aa in std::__throw_out_of_range_fmt (__fmt=<optimized out>) from /usr/lib/libstdc++.so.6
#2  0x00005555556e1aa0 in std::vector<CommandSequenceUnit, std::allocator<CommandSequenceUnit> >::_M_range_check (
    this=0x55555598d958, __n=65535) at /usr/include/c++/8.2.1/bits/stl_vector.h:960
#3  0x00005555556e12a9 in std::vector<CommandSequenceUnit, std::allocator<CommandSequenceUnit> >::at (
    this=0x55555598d958, __n=65535) at /usr/include/c++/8.2.1/bits/stl_vector.h:981
#4  0x00005555556e011d in CommandSequence::getSequenceTypeAt (this=0x55555598d920, n=65535)
    at ../BambooTracker/instrument/command_sequence.cpp:47
#5  0x00005555556e06c3 in CommandSequence::Iterator::getCommandType (this=0x555559a25070)
    at ../BambooTracker/instrument/command_sequence.cpp:177
#6  0x000055555565a04e in OPNAController::writeEnvelopeSSGToRegister (this=0x555555b26660, ch=0, seqPos=65535)
    at ../BambooTracker/opna_controller.cpp:2707
#7  0x0000555555657744 in OPNAController::releaseStartSSGSequences (this=0x555555b26660, ch=0)
    at ../BambooTracker/opna_controller.cpp:2050
#8  0x00005555556558b5 in OPNAController::keyOffSSG (this=0x555555b26660, ch=0, isJam=true)
    at ../BambooTracker/opna_controller.cpp:1742
#9  0x00005555555ce3c8 in BambooTracker::jamKeyOff (this=0x555555966780, key=JamKey::LOW_C)
    at ../BambooTracker/bamboo_tracker.cpp:751
#10 0x000055555558767e in MainWindow::keyReleaseEvent (this=0x555555966040, event=0x7fffffffdcf0)
    at ../BambooTracker/gui/mainwindow.cpp:554
```

**Comment by rerrahkr** [comment]
It caused by a wrong release point of the module like #59. I replaced it with fixed one at 71034a3cc0f5b4320c5325586ef256364a1c916d.

**Comment by jpcima** [comment]
I understand. Thank you for fixing.
